### ISSUE TITLE
PIV Results page feedback

### ISSUE THREAD

**Issue by kp-121** [mI-PIV/app#121]
Testing feedback suggestions:

 [x] - Distinct background colors for result sections
 [x] - Hide toggles/widgets for sections that aren't currently displayed 
 [ ] - Add lightbulbs (with links to respective curricular modules) to result sections
 [x] - Make sure the 'save image' popup is accurate and current regarding the saved images location on phone.

**Comment by kp-121** [comment]
#125 adds collapsible sections and color-coded sections to the results page

**Comment by jack-elliott** [comment]
Waiting on light bulb content, closing this and adding that to the excel sheet of desired features. Will open a new issue when content is prepared
